src/widget/wlibrarytableview.cpp

```diff
@@ -53,37 +55,9 @@ WLibraryTableView::WLibraryTableView(QWidget* parent,
 }
 
 WLibraryTableView::~WLibraryTableView() {
+    m_modelStateCache.clear();
```

Comment:
This is redundant as the destructor is invoked immediately after.

Reply:
true, I adopted this detail without questioning.. : \
removed.